packages/cli/src/tests/utils/changelog.js

```diff
@@ -27,25 +32,64 @@ describe('changelog utils', () => {
     });
 
     it('should find changelog for 1.0.0', () =>
-      changelog.getVersionChangelog('1.0.0', appDir).then((log) => {
-        log.should.eql('* Removing beta "label".\n* Minor docs fixes.');
+      changelogUtil.getVersionChangelog('1.0.0', appDir).then((log) => {
+        log.changelog.should.eql(
+          '* Removing beta "label".\n* Minor docs fixes.'
+        );
       }));
 
     it('should not find changelog for 2.3.0', () =>
-      changelog.getVersionChangelog('2.3.0', appDir).then((log) => {
-        log.should.eql('');
+      changelogUtil.getVersionChangelog('2.3.0', appDir).then((log) => {
+        log.changelog.should.eql('');
       }));
 
     it('should not fail when there is no changelog file', () =>
-      changelog
+      changelogUtil
         .getVersionChangelog('1.0.0', './unexistent-directory')
         .then((log) => {
-          log.should.eql('');
+          log.changelog.should.eql('');
         }));
 
     it('should not fail when versions follow "different" formatting', () =>
-      changelog.getVersionChangelog('2.0.0', appDir).then((log) => {
-        log.should.eql('* Fix some bugs.\n* Major docs fixes.');
+      changelogUtil.getVersionChangelog('2.0.0', appDir).then((log) => {
+        log.changelog.should.eql('* Fix some bugs.\n* Major docs fixes.');
       }));
```

Comment:
Can we add another test for [PDE-3820](https://zapierorg.atlassian.net/browse/PDE-3820)?

First, add a `#### 0.0.1` section to the test data:

```javascript
    before(() => {
      // Create a temp CHANGELOG.md
      appDir = makeTempDir();
      fs.writeFileSync(
        path.join(appDir, 'CHANGELOG.md'),
        '## 3.0.0\n\n' +
          'Made some changes that affect app actions\n' +
          '- Update the trigger/pr_review action, as well as changes for #456\n' +
          'However, we also addressed fixed open issues!\n' +
          '- Fix #123 and an issue with create/send_message\n\n' +
          '## new and improved version! (v2.0.0)\n\n' +
          '* Fix some bugs.\n' +
          '* Major docs fixes.\n\n' +
          '## 1.0.0\n\n' +
          '* Removing beta "label".\n' +
          '* Minor docs fixes.\n' +
          '#### 0.0.1\n' +
          'initial release\n\n' +
          'just for internal testing\n\n'
      );
    });
```

Then add a new test case:

```javascript

    it('should be forgiving on the markdown format', async () => {
      const { appMetadata, issueMetadata, changelog } =
        await changelogUtil.getVersionChangelog('0.0.1', appDir);
      should(appMetadata).equal(undefined);
      should(issueMetadata).equal(undefined);
      changelog.should.equal('initial release\n\njust for internal testing');
    });
```